splat.yaml

```diff
@@ -356,53 +358,23 @@ segments:
     - [0x568D0, c]
     - [0x574F0, c]
     - [0x57580, .data, 46270]
-    - [0x579D0, .rodata, 46270]
-    - [0x57A70, rodata]
-    - [0x57AA0, rodata]
-    - [0x59F30, rodata]
-    - [0x5A180, rodata]
-    - [0x5A400, rodata]
-    - [0x5A430, rodata]
-    - [0x5A490, rodata]
-    - [0x5A4F0, rodata]
-    - [0x5A530, rodata]
-    - [0x5A570, rodata]
-    - [0x5A580, rodata]
-    - [0x5A610, rodata]
-    - [0x5AA00, rodata]
-    - [0x5AD50, rodata]
-    - [0x5AD80, rodata]
-    - [0x5AD90, rodata] # I believe rodata actually starts here (aside from the one blob at 0x579D0 - 0x57A70)
+    - { start: 0x579D0, type: .rodata, name: 46270, linker_section_order: .data }
+    - [0x57A70, data]
+
+    - [0x5AD90, rodata]
```

Comment:
The strings from 0x5AD90 and are used in arrays and didn't want to mess with those. That's why they don't have a corresponding C file yet